binance_alarm/settings.py

```diff
@@ -83,8 +84,19 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#databases
 
 # https://docs.djangoproject.com/en/dev/ref/settings/#databases
-DATABASES = {"default": env.db("DATABASE_URL")}
-DATABASES["default"]["ATOMIC_REQUESTS"] = True
+
```

Comment:
Get used to DATABASE_URL. You should use the best solution, even if takes time to understand it.

Reply:
++